components/indicators/IndicatorValueSummary.js

```diff
@@ -52,7 +52,14 @@ const ChangeSymbol = styled.span`
     `${props.theme.fontFamilyTiny}, ${props.theme.fontFamilyFallback}`};
 `;
 
-function determineDesirableDirection(values, goals) {
+function determineDesirableDirection(desiredTrend, values, goals) {
```

Comment:
Since this function is only called once below, I don't see much use for it, since I think `INCREASING` is a bit more descriptive than `+`anyway.

Reply:
Especially since the `values` and `goals` are not used.

Reply:
The default value for desiredTrend is `A_` (Automatically detect desired trend) so basically this provides the default fall back solution if no desiredTrend is set. I shall comment it in the code as `A_` is not immediately obvious in ui code